src/aboutPage/AboutPage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import classNames from 'classnames';
+
+import styles from './AboutPage.module.css';
+import AboutPageFi from './AboutPageFi';
+import AboutPageSv from './AboutPageSv';
+import AboutPageEn from './AboutPageEn';
+import commonContentStyles from '../common/cssHelpers/content.module.css';
+import PageLayout from '../common/pageLayout/PageLayout';
+
+function AboutPage(): React.ReactElement {
```

Comment:
Looks like Sonar is complaining about test coverage on these about pages. There isn't much to test, could add simple tests just to see that everything renders correctly. 🙂